src/components/RefinementList/RefinementList.tsx

```diff
@@ -23,7 +24,7 @@ export type RefinementListProps<
   createURL: CreateURL<string>;
   cssClasses: TCSSClasses;
   depth?: number;
-  facetValues?: TRefinementListItem[];
+  facetValues?: TRefinementListItem[] | HierarchicalMenuItem[];
```

Comment:
why does one have a T prefix and not the other?

Reply:
I has to rename the `RefinementListItem` type to `TRefinementListItem` since there is already the `RefinementListItem` component in this file. I did not rename the `HierarchicalMenuItem` type since there is no name conflict with it. I can apply the same naming to it to be more coherent.